spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -185,9 +177,66 @@ public Object run() {
 		return null;
 	}
 
+	protected PoolingHttpClientConnectionManager newConnectionManager() {
```

Comment:
It's not clearer, IMO it's worse as I can't tell what changes (if any) you made to this method.